src/modules/confirmation/ModuleConfirmation.vue

```diff
@@ -528,6 +556,7 @@ export default {
     EventBus.$on(
       EventNames.SHOW_BATCH_TX_MODAL,
       async (arr, resolver, isHardware) => {
+        console.log('supposed to be here', arr);
```

Comment:
console! unless its supposed to be here lolol

Reply:
consooooolleeeeee